src/frontend/src/common/components/library/data_subview/components/DownloadModal/index.tsx

```diff
@@ -246,6 +251,29 @@ const DownloadModal = ({
                   </CheckboxLabel>
                 </StyledFileTypeItem>
               )}
+              {isPrepFilesFlagOn && (
```

Comment:
do you have to use another feature flag block here, or could they be combined into one?

Reply:
It wanted a single element child so I did this rather than using a fragment.  The thinking is that this is less likely to get accidentally left in the code after the flag is no longer needed.